package.json

```diff
@@ -61,6 +64,7 @@
     "@angular/language-service": "17.3.2",
     "@commitlint/cli": "^19.0.0",
     "@commitlint/config-angular": "^19.0.0",
+    "@jscutlery/playwright-ct-angular": "link:dist/packages/playwright-ct-angular",
```

Comment:
Why is it needed?

Reply:
Nice catch 😉
I didn't know where to put a comment for this 😅

The thing is that we pass the module to playwright which passes it to babel: 
https://github.com/jscutlery/devkit/pull/358/files#diff-505bc1cdc622f47209d94b9bd0b2e47fb4363b1bb09e80641df1127e6cf74cb2R75

This targets a secondary entrypoint in the generated package.json instead of hardcoding either the path to the cjs or esm js file (`transform-angular.cjs.js` etc...).
In addition to this, this makes sure that everything is working with the built version of our package.

That is why the project at `tests/playwright-ct-angular-wide` depends on the `build` target of the package.